### DISCUSSION TITLE
Welcome to Filter-Rubika-Ryson Discussions!

### DISCUSSION THREAD

**Discussion by FILTERING-RUBIKA-RYSON** [FILTERING-RUBIKA-RYSON/Filter-Rubika-Ryson#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by server-python** [comment]
عجب:)
داش بخش sms bomber مشکل داره:|
بخش کد هات هم که فیکن:|
منم اصلا میشناسی؟
سرور اسکریپت:|

تو روبیکا اکم فیلتر شد
@server_script
آیدی روبیکامه:(
